README.md

```diff
@@ -60,3 +60,49 @@ Run `docker-compose run composer require wpackagist-plugin/plugin-name`.
+        Port <see "SFTP/SSH" details for Staging Environment>
+        IdentityFile <Location on your local machine of the SSH key>
+        ForwardAgent yes
+    host <your site>_live
+        User <see "SFTP/SSH" details for Live Environment>
+        Hostname <see "SFTP/SSH" details for Live Environment>
+        Port <see "SFTP/SSH" details for Live Environment>
+        IdentityFile <Location on your local machine of the SSH key>
+        ForwardAgent yes
+   ```
+6. Add you GitHub SSH key to `ssh-agent` [following these instructions](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent). The command will be something like `ssh-add ~/.ssh/gitlab_key_rsa`
+7. We are going to begin with the staging environment. SSH onto Kinsta staging environment using this command `ssh nurses_united_staging`.
+8. Test GitHub works by running the command `ssh -T git@github.com`. After accepting the authenticity of the host you should see a friendly message from GitHub.
+9. Remove the `public` directory with `rm -r public`. Clone the code into the `public/` directiory: `git clone git@gitlab.com:commonknowledge/nurses-united/website.git public`.
+10. `cd public; composer install`.
+11 Return to the `public` directory, `cd ~/public`. If this is the first time, copy `.env.example` to `.env` with `cp .env.example .env` and modify details appropriately [as per the Bedrock documentation](https://roots.io/bedrock/docs/environment-variables/). You can use Vim on the server. The details of the database are on the Kinsta admin panel under the site itself then "Database Access". The `WP_HOME` can be `http://localhost`.
+12. You need to create database tables for WordPress. Run `wp core install --url=<URL from Primary domain in Kinsta admin panel> --title=<site name> --admin_user=<desired username> --admin_email=<desired password>`. This will output the password for the user you have just created to the terminal. Save it for when you need to login.
+13. Ask Kinsta to update NGINX to point at `public/web` on Intercom chat inside the Kinsta control panel. Note, not `public/current/web`, which is the directory if you are deploy Bedrock with [Trellis](https://github.com/roots/trellis). This installation does not use Trellis.
+14. You can now point the domains to this installation of WordPress following [Kinsta's instructions](https://kinsta.com/knowledgebase/add-domain/).
+15. Head to Tools for the site in the staging environment. Use "SSL certificate" to generate a new Let's Encrypt SSL certificate and wait for this to complete.
+16. Still on the tools page, setup Force HTTPS by clicking on Modify and dollowing your nose, selecting "Force all traffic to the primary domain" along the way. Edit the `.env` file created in step 12, to have `WP_HOME` include `https` not `http`.
+17. Finally on the tools page upgrade to PHP 7.4 by changing the "PHP Engine" dropdown.
```

Comment:
Kinsta only supports PHP 8+ nowadays, according to their customer support.

Reply:
Removed this for the moment.